src/annotations/components/controlBar/AnnotationsControlBar.tsx

```diff
@@ -37,11 +37,14 @@ export const AnnotationsControlBar: FC = () => {
   const dispatch = useDispatch()
 
   const changeWriteMode = () => {
+    event('dashboard.annotations.change_write_mode.toggle', {
```

Comment:
this is the key that will show up in query builder

here's how it'll look in a script:
![Screen Shot 2021-03-15 at 5 36 47 PM](https://user-images.githubusercontent.com/146112/111238781-08e91580-85b5-11eb-920e-f5d3cb8556a4.png)